source/Ansible-RabbitMQ/RabbitMQPublisher.class.st

```diff
@@ -61,30 +68,34 @@ RabbitMQPublisher >> onPublicationConfirmationDo: anAckBlock onRejectionDo: aNac
       theChannel onPublicationConfirmationDo: anAckBlock onRejectionDo: aNackBlock ]
 ]
 
-{ #category : 'private - configuring' }
-RabbitMQPublisher >> persistentDeliveryMode [
-
-  ^ connection protocolClass basicPropertiesClass new deliveryMode: 2
-]
-
 { #category : 'publishing' }
-RabbitMQPublisher >> publish: aMessageCollection onQueueNamed: aQueueName [
+RabbitMQPublisher >> publish: aMessage toExchange: anExchangeName usingRoutingKey: aRoutingKey [
```

Comment:
The main issue here is that I would keep the `named:` suffix, , like in `toExchangeNamed:`,  if you want to make explicit that you're collaborating with a name. But it may be worth to focus on the role, abandoning the name and key suffixes. The message then would be `publish: aMessage to: anExchange routedTo: aRoute`,  `publish: aMessage to: anExchange through: aRoute` or `publish: aMessage to: anExchange following: aRoute`.